src/main/avro/need.avdl

```diff
@@ -0,0 +1,14 @@
+@namespace("no.nav.dagpenger.events.avro.need")
+protocol DagpengerNeed {
+  record Dokument {
+    string dokumentId;
+    string navSkjemaId;
+  }
+
+  record Need {
```

Comment:
not sure if "need" is a good name here. What does domain-expert calls it? "Krav"?